domain/src/main/kotlin/com/krzykrucz/elesson/currentlesson/domain/topic/domain/Topic.kt

```diff
@@ -0,0 +1,14 @@
+package com.krzykrucz.elesson.currentlesson.domain.topic.domain
+
+import com.krzykrucz.elesson.currentlesson.domain.NaturalNumber
+import com.krzykrucz.elesson.currentlesson.domain.NonEmptyText
+import com.krzykrucz.elesson.currentlesson.domain.attendance.CheckedAttendance
+import com.krzykrucz.elesson.currentlesson.domain.startlesson.LessonIdentifier
+import java.time.LocalDate
+
+data class LessonOrdinalNumber(val number: NaturalNumber)
+data class FinishedLessonsCount(val count: NaturalNumber)
+data class TopicTitle(val title: NonEmptyText)
+data class LessonTopic(val lessonOrdinalNumber: LessonOrdinalNumber, val topicTitle: TopicTitle, val date: LocalDate)
+data class InProgressLesson(val lessonIdentifier: LessonIdentifier, val lessonTopic: LessonTopic, val attendance: CheckedAttendance)
```

Comment:
in reference to Business Terms from the Design Event Storming,`InProgressLesson` type consists of `attendance`, `topic`, `class`. Here I can see  `LessonIdentifier` which is not obvious for me as is not defined anywhere in diagrams. Aren't `StartedLesson` `LessonTopic` and `CheckedAttendance` parameters here?

Reply:
there might be errors in the event storming. We don't need to pass CheckedAttendance or StartedLesson if we don't use them as a whole. I would just pass the subproperties of the mentioned types, the ones which are really needed.